src/quo2/components/settings/settings_item/style.cljs

```diff
@@ -20,9 +22,10 @@
    :align-items    (or align-action :center)})
 
 (def left-container
-  {:margin-left     12
-   :height          "100%"
-   :justify-content :flex-start})
+  {:margin-horizontal 12
+   :flex              1
```

Comment:
can you give a quick check of other uses of this component to ensure it's not breaking its uses 👍

Reply:
checked it, all fine 👍

Reply:
🙌